lib/ddtrace/tracer.rb

```diff
@@ -161,6 +161,13 @@ def active_span
       @buffer.get()
     end
 
+    # stats returns a dictionary of stats about the writer.
```

Comment:
I did this only for the sake of integration testing, but then I could probably implement this only in tests. Will take a look, thanks for pointing it out.